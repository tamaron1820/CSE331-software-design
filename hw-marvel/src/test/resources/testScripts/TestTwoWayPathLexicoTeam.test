# Tests that if player(In Marvel's example the character) have many team
# priority increases in lexicographical order.

# To run this test, the marvel dataset must exist at
#   hw-marvel/src/main/resources/data/soccerPlayer.csv

LoadGraph soccer soccerPlayer.csv

FindPath soccer Messi Robinho

FindPath soccer Robinho Messi