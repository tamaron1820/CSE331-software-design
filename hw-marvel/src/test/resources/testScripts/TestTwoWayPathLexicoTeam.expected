# Tests that if player(In Marvel's example the character) have many team
# priority increases in lexicographical order.

# To run this test, the marvel dataset must exist at
#   hw-marvel/src/main/resources/data/soccerPlayer.csv

loaded graph soccer

path from Messi to Robinho:
Messi to Neymar via Barcelona
Neymar to Robinho via Santus

path from Robinho to Messi:
Robinho to Neymar via Santus
Neymar to Messi via Barcelona